tests/imgls

```diff
@@ -1,23 +1,166 @@
-#!/bin/bash
-if [ $# -eq 0 ]; then
-  echo "Usage: image.sh filename [filename...]"
-  exit 1
-fi
-for fn in "$@"
-do
-  dims=$(php -r '$a = getimagesize("$argv[1]"); if ($a==FALSE) exit(1); else { echo $a[0] . "x" .$a[1]; exit(0); }' "$fn")
-  rc=$?
-  if [[ $rc == 0 ]] ; then
-    printf '\033]1337;File=name='`echo -n "$fn" | base64`";"
-    wc -c "$fn" | awk '{printf "size=%d",$1}'
-    printf ";inline=1;height=3;width=3;preserveAspectRatio=true"
-    printf ":"
-    base64 < "$fn"
-    printf '\a'
-    printf '\033[A'
-    echo -n "$dims "
-    ls -l "$fn"
-  else
-    ls -l "$fn"
-  fi
-done
+#!/usr/bin/perl
```

Comment:
As I mentioned earlier, I'd like to keep both versions around (the simple reference version and the fancy do-everything perl version). So please give this one a different name.